python/metatensor_torch/tests/atomistic/model.py

```diff
@@ -29,16 +31,16 @@ def forward(
         outputs: Dict[str, ModelOutput],
         selected_atoms: Optional[Labels] = None,
     ) -> Dict[str, TensorMap]:
-        if "dummy" in outputs:
+        if "tests::dummy::long" in outputs:
             block = TensorBlock(
-                values=torch.tensor([[0.0]]),
+                values=torch.tensor([[0.0]], dtype=torch.float64),
```

Comment:
why?

Reply:
Because the model capabilities are registered as float64. With check_consistency, an error is raised if the output doesn't match the declared dtype of the model. Since predictions of this model weren't being called in any test, this bug wasn't showing

Reply:
Same I guess, it was declared as float64 below?

Reply:
Yes, correct